pkg/admission/validator/shoot.go

```diff
@@ -44,8 +44,9 @@ var (
 
 // shoot validates shoots
 type shoot struct {
-	decoder runtime.Decoder
-	Logger  logr.Logger
+	decoder        runtime.Decoder
+	lenientDecoder runtime.Decoder
+	Logger         logr.Logger
```

Comment:
Is this Logger used somewhere? Why is it exported?

Reply:
No, it's not, perhaps some leftover. It has nothing to do with my change, but I will delete it anyway.

Reply:
Done

Reply:
Ah, I missed that it wasn't even added by you. Anyways, thanks for cleaning up!